### DISCUSSION TITLE
What to put for ssl?

### DISCUSSION THREAD

**Discussion by lucyholic** [beyondcode/laravel-websockets#952] [Q&A]
Might be a dumb question, but it would be great if any one can help.
I'm using Let's Encrypt and wondering if I've put the right path for local cert and pk.

local_cert: ssl.mydomain.crt
local_pk: ssl.mydomain.key

Does it look correct?

Thank you in advance!

**Comment by fredsal** [comment]
Put it on `.env`
https://github.com/beyondcode/laravel-websockets/blob/605a7fa71ddabfffe381b079acc21403c7547ea2/config/websockets.php#L240-L254

**Reply by lucyholic** [reply]
Hi, I should've been more clear with my question. I was wondering if I put the correct extensions. I have a bundle of ssl files, that end with pen, crt, key, and ca.
